velox/functions/prestosql/aggregates/ArrayAggAggregate.cpp

```diff
@@ -143,6 +208,25 @@ class ArrayAggAggregate : public exec::Aggregate {
     });
   }
 
+  bool supportAddRawClusteredInput() const override {
+    return !ignoreNulls_;
```

Comment:
Would it be possible to document this limitation somewhere? PR description and a comment here explaining why this is not supported.

Reply:
Let me see if I can remove this limitation along with the one on aggregate mask